stackoverflow_survey_2024 (#737)
rfordatascience/tidytuesday: data/curated/stackoverflow_survey_2024/stackoverflow_survey_single_response.md

| variable              | class     | description                                                                         |
|:--------------------------|:-----------------|:--------------------------|
| response_id           | double    | Respondent ID                                                                       |

> | main_branch           | integer   | Which of the following options best describes you today?                            |

Reviewer: ```suggestion
| main_branch           | integer   | Professional coding level of the respondent |
```

| age                   | integer   | Age                                                                                 |
| remote_work           | integer   | Current work situation                                                              |
| ed_level              | integer   | Highest education level completed                                                   |
| years_code            | integer   | Years have you been coding in total; More than 50 years coded as 51                 |
| years_code_pro        | integer   | Years have you coded professionally; More than 50 years coded as 51                 |
| dev_type              | integer   | Best current-job description                                                        |
| org_size              | integer   | People in the organization                                                          |

> | purchase_influence    | integer   | Level of influence in purchasing new technology at your organization                |

Reviewer: ```suggestion
| purchase_influence    | integer   | Level of influence in purchasing new technology at their organization                |
```

> | buildvs_buy           | integer   | When asked to provide a recommendation, what has been your most recent choice?      |

Reviewer: ```suggestion
| buildvs_buy           | integer   | How much customization was needed in most recent tool recommendation |
```

> | country               | character | Country Living In                                                                   |

Reviewer: ```suggestion
| country               | character | Country in which the respondent lives |
```

| currency              | character | Currency of the country                                                             |
| comp_total            | double    | Total compensation                                                                  |
| so_visit_freq         | integer   | Stack Overflow visiting frequency                                                   |

> | so_account            | integer   | Stack Overflow Account Status                                                       |

Reviewer: ```suggestion
| so_account            | integer   | Stack Overflow account status                                                       |
```

| so_part_freq          | integer   | Stack Overflow participation frequency                                              |

> | so_comm               | integer   | Do you consider yourself a member of the Stack Overflow community?                  |

Reviewer: ```suggestion
| so_comm               | integer   | Whether the respondent considers themself a member of the Stack Overflow community?                  |
```

| ai_select             | integer   | Use AI in development process                                                       |
| ai_sent               | integer   | Stance on using AI tools as part of development workflow                            |
| ai_acc                | integer   | Trust in accuracy of AI as part of development workflow                             |

> | ai_complex            | integer   | How well do the AI tools you use in your development workflow handle complex tasks? |

Reviewer: ```suggestion
| ai_complex            | integer   | How well the respondent believes the AI tools they use in development workflows handle complex tasks |
```

> | ai_threat             | integer   | Do you believe AI is a threat to your current job?                                  |
> | survey_length         | integer   | How do you feel about the length of the survey this year?                           |
> | survey_ease           | integer   | How easy or difficult was this survey to complete?                                  |

Reviewer: ```suggestion
| ai_threat             | integer   | Belief that AI is a threat to current job |
| survey_length         | integer   | Feeling about the length of the Stack Overflow Developer Survey this year |
| survey_ease           | integer   | Ease of completion of this survey |
```
